Samples/ObservableCounter/Sources/CounterListView.swift

```diff
@@ -0,0 +1,23 @@
+import SwiftUI
```

Comment:
commenting here primarily for threading purposes, but wondering if this issue is specific to our setup, or can something analogous be replicated with 'vanilla' SwiftuI?

Reply:
Specific to our setup. There may be some assumptions built around the state being a reference type that we're running into.